index.html

```diff
@@ -0,0 +1,141 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo"><a href="#">My Logo</a></div>
```

Comment:
- **[OPTIONAL]** Since you added the `<a>` tag, now your logo style has changed slightly and needs some adjustment 👇🏽.

**How it looks now:**
![image](https://user-images.githubusercontent.com/43172164/198195796-f07e8b80-4a49-4835-97fe-147aa616c625.png)

**How it looked before:**
![image](https://user-images.githubusercontent.com/43172164/198195887-8b8c9e20-f636-496a-95bc-305fa8aa1ff5.png)

Reply:
Ok, thanks @Yazino12